src/torchaudio/transforms/_transforms.py

```diff
@@ -1846,7 +1847,7 @@ def forward(
             Tensor: Loss with the reduction option applied. If ``reduction`` is  ``"none"``, then size (batch),
             otherwise scalar.
         """
-        return F.rnnt_loss(
+        return rnnt_loss(
```

Comment:
Can you share why this change was needed?

Reply:
Functions that get wrapped with a decorator confuse torchscript. There's a bunch of tests for torchscript functionality in the repository, and torchscript isn't officially deprecated yet. Probably the best solution would be to fix this bug in torchscript. But the easier solution is to avoid using the wrapped versions when possible. In this case, because we give a deprecation warning on initialization of the transform, we don't need to give one again when the transform gets called. And this way, the initialization (which is wrapped) takes place outside of torchscript's tracing, while the running part (which torchscript *does* trace) doesn't have any wrapping.